smarts/sstudio/genhistories.py

```diff
@@ -42,8 +43,10 @@
 
 try:
     from waymo_open_dataset.protos import scenario_pb2  # pytype: disable=import-error
+    from waymo_open_dataset.protos.map_pb2 import (
+        TrafficSignalLaneState,
+    )  # pytype: disable=import-error
```

Comment:
It is surprising this is needed inside a catch for import error. I could suggest wrapping the import block.
```suggestion
    # pytype: disable=import-error
    from waymo_open_dataset.protos import scenario_pb2  
    from waymo_open_dataset.protos.map_pb2 import (
        TrafficSignalLaneState,
    )
    # pytype: enable=import-error
```